examples/examples/custom_type_derives.rs

```diff
@@ -20,7 +20,7 @@
     runtime_metadata_path = "examples/polkadot_metadata.scale",
     // We can add (certain) custom derives to the generated types by providing
```

Comment:
Curious: do you happen to know what this "(certain)" means? Do we know which ones work and which don't?